src/megameklab/com/util/StringUtils.java

```diff
@@ -356,8 +260,8 @@ public static String getEquipmentInfo(Entity unit, Mounted mount) {
         return info.trim();
     }
 
-    public static String getEquipmentInfo(Aero unit, Mounted mount) {
-        String info = "";
+    public static String getAeroEquipmentInfo(Mounted mount) {
```

Comment:
The unit parameter was to distinguish this method from getEquipmentInfo(Entity, Mounted), but not actually used. The overloaded method required casting Entity to Aero to get the right one.